### PR TITLE
Add support for valgrind outputs filtered by CTest/CDash

### PR DESCRIPTION
**Pull request by LucHermitte**
This is a first PR.

 I'll eventually have other(s?) that move the functions to autoload plugins, and that provide options to chose the keybindings (a mapping on `<space>` with this new trend to set the leader to `<space>` can be problematic).

### CODE REVIEW

**Review by LAHumphreys — APPROVED** [review]
Thanks, I've merged the function bangs in.

File: plugin/valgrind.vim [hunk 1]
```diff
@@ -123,8 +123,8 @@ function! s:Valgrind( ... )
     else
         silent execute 'split '.l:tmpfile
     endif
-    silent execute 'g!/^==\d*==/d'
-    silent execute '%s/^==\d*== //e'
+    silent execute 'g!/^\(\d\+: \)\===\d*==/d'
```
Comment (LAHumphreys) [inline plugin/valgrind.vim]: I agree the use case is valid, but I'm not sure hard-coding the optional prefix here is the right answer - would we be better off having a prefix variable which can be configured in a vimrc?

**Review by LAHumphreys — CHANGES_REQUESTED** [review]

File: plugin/valgrind.vim (hunk 1 above)
Comment (LucHermitte) [reply plugin/valgrind.vim]: Either that or a hook that removes noise. For instance, I'm thinking of a possible encapsulation of each line by an XML node. In that case, we'll need to pre-process the lines differently.

**Review by LAHumphreys — APPROVED** [review]

**Comment by LAHumphreys** [comment]
Nice - thanks, I've pulled this in :)

I agree with you that making these key binding configurable would be valuable.

**Pull request closed without merging**